tests/string/HslColor.test.ts

```diff
@@ -0,0 +1,133 @@
+import * as RA from 'fp-ts/ReadonlyArray'
+import * as E from 'fp-ts/Either'
+
+import { pipe, tuple } from 'fp-ts/function'
+
+import * as HslColor from '../../src/string/HslColor'
+
+import { cat, combineExpected, validateArbitrary } from '../../test-utils'
+
+const validStrings = [
+  'hsl(360,0000000000100%,000000100%)',
+  'hsl(000010, 00000000001%, 00000040%)',
+  'HSL(00000,0000000000100%,000000100%)',
+  'hsL(0, 0%, 0%)',
+  'hSl(  360  , 100%  , 100%   )',
+  'Hsl(  00150  , 000099%  , 01%   )',
+  'hsl(01080, 03%, 4%)',
+  'hsl(-540, 03%, 4%)',
+  'hsla(+540, 03%, 4%)',
+  'hsla(+540, 03%, 4%, 500)',
+  'hsl(+540deg, 03%, 4%, 500)',
+  'hsl(+540gRaD, 03%, 4%, 500)',
+  'hsl(+540.01e-98rad, 03%, 4%, 500)',
+  'hsl(-540.5turn, 03%, 4%, 500)',
+  'hsl(+540, 03%, 4%, 500e-01)',
+  'hsl(+540, 03%, 4%, 500e+80)',
+  'hsl(4.71239rad, 60%, 70%)',
+  'hsl(270deg, 60%, 70%)',
+  'hsl(200, +.1%, 62%, 1)',
+  'hsl(270 60% 70%)',
+  'hsl(200, +.1e-9%, 62e10%, 1)',
+  'hsl(.75turn, 60%, 70%)',
+  // 'hsl(200grad+.1%62%/1)', //supposed to pass, but need to handle delimiters
```

Comment:
💬  Is this just strange because it has no spaces or commas?

Reply:
Yeah, you can use `+` as a delimiter 🤮 

```
hsl(50+80%40%/.6);
```